### DISCUSSION TITLE
PyMISP search events by galaxy/ clusters

### DISCUSSION THREAD

**Discussion by amitw15** [MISP/PyMISP#878] [Q&A]
Hi!
Recently I tried to filter using the cluster/ galaxy methods, but I didn’t find a way to do that.
My goal is to look for events that has certain galaxies or clusters related to them,
Ill appreciate your help

**Comment by Rafiot** [comment]
You can search galaxies the same way you search tags, it will look something like that: `misp-galaxy:360net-threat-actor="Lazarus - APT-C-26"`.

Please let us know if it answers your question.

**Reply by amitw15** [reply]
Thanks for your reply!
Actually what I meant was to look for all the events that have some galaxies/ clusters.
For example: Im interested In all the events that have been published in last week and Galaxy: 'Sector' has been related to.
If im trying to do it like: misp.search(contoller='events',galaxy='sector' etc..) Im getting a lot of results, most of them dont have any relation to the galaxy 'sector'.
